README.md

```diff
@@ -25,6 +25,49 @@ If you want to download regularly updated builds for Android, Windows x86 and x6
 
 For game compatibility, see [community compatibility feedback](https://report.ppsspp.org/games).
 
+What's new in 1.13
+==================
+
+General
+
+* Fix assorted Android "scoped storage"-related bugs and performance issues ([#15237]), ([#15487]), etc.
+* Analog mapping for fast-forward ([#15645])
+* Major softgpu accuracy fixes and speedups, including a JIT ([#15163]), ([#15345]), ([#15389]), ([#15529]), ([#15440]), ([#15410]), ([#15405]), ([#15400]) and many, many more
+* Fixed some NEON code paths ([#15481])
+* Fix performance of texture uploads with Vulkan ([#15474])
+* Don't include the large font atlas when we don't need it
+* Improved upscaling shaders ([#15566])
+* Vulkan texture upscaling performance improvements ([#15238]), etc.
+* Vulkan correctness fixes ([#15217]), ([#15211]), use the VMA allocator ([#15162]), etc.
+* Fixes to depth culling ([#15106]), many more
+* Background loading of texture replacement
+* Threading manager improvements and fixes ([#15470]), etc.
+* Added search in settings ([#14414])
+* Added fast button repeats on custom touch buttons ([#15613])
+* Two new bicubic upscaling shader: Catmull-Rom and Mitchell-Netravali ([#15569])
+* Allow to change screen rotation per game and to bind a key to change it ([#15494], [#15510])
+* Re-enabled software rendering option on Android ([#12958])
+
+Game fixes
+
+* Add more workarounds for Mali driver bugs ([#15016])
+* Vortex in God of War: Ghost of Sparta can now be passed ([#15640])
+* Various proAdhoc fixes ([#15213]), ([#15215]), and many more
+* Correct flickering text in Sol Trigger and Last Ranker. ([#15549])
+* Fix and improve line drawing in Echochrome ([#15583]), after line refactoring ([#15073]), ([#15075]), ([#15075])
```

Comment:
I think Saramagrean was talking about this line too as 15075 appears twice in the list.

Reply:
and Background loading of texture replacement is should https://github.com/hrydgard/ppsspp/pull/15025 ,right?

Reply:
Yeah, that's right, I'll fix both things.